srcalc/src/main/java/gov/va/med/srcalc/domain/model/ValueMatcher.java

```diff
@@ -17,16 +17,19 @@
  * Expression Language (SPeL) expression.
  */
 @Embeddable
-public class ValueMatcher
+public final class ValueMatcher
 {
     private Variable fVariable;
     private Expression fBooleanExpression;
+    private boolean fEnabled;
     
     /**
 	 * Mainly intended for reflection-based construction.
 	 */
     ValueMatcher()
-    {    	
+    {
+        // Set the summand expression to avoid a NullPointerException from Hibernate.
```

Comment:
I don't like the mysterious "to avoid a NullPointerException from Hibernate" explanation here. It's clearer to say "to avoid a NullPointerException if getBooleanExpression() is called before setBooleanExpression()."